widgetastic_manageiq/__init__.py

```diff
@@ -4274,12 +4274,37 @@ def schedule_migration(self, plan_name, cancel=False, after_mins=1):
         try:
             el = self._get_plan_element(plan_name)
             schedule_button = self.browser.element(self.ITEM_SCHEDULE_BUTTON_LOCATOR, parent=el)
+            wait_for(
+                lambda: schedule_button.is_enabled(),
+                delay=5,
+                num_sec=30,
+                message="waiting for schedule button to be enabled.",
+            )
             schedule_button.click()
             if schedule_button.text == "Schedule" and not cancel:
                 schedule_time_input = self.root_browser.element(self.ITEM_SCHEDULE_INPUT_LOCATOR)
                 schedule_time_input.click()
-                for i in range(after_mins):
-                    self.root_browser.click(self.ITEM_MODAL_MINUTE_INCREMENT_LOCATOR)
+                try:
+                    for i in range(after_mins):
+                        self.root_browser.click(self.ITEM_MODAL_MINUTE_INCREMENT_LOCATOR)
+                except NoSuchElementException:  # Throws this in FF browser
+                    self.logger.info(
+                        "Schedule using dateTimePicker failed, using datetime and timedelta."
+                    )
+                    from datetime import datetime, timedelta
+
+                    current_time = datetime.strptime(
+                        schedule_time_input.get_attribute("value"), "%m/%d/%Y %I:%M %p"
+                    )
+                    schedule_time_input.clear()
+                    schedule_time_input.send_keys(
+                        datetime.strftime(
+                            current_time + timedelta(seconds=60 * after_mins), "%m/%d/%Y %I:%M %p"
```

Comment:
Required:
No need to convert `minutes` into `seconds`. Its core python powerfull implementation you can directly go with `minutes` tag; lib will take care of it. 
```
In [7]: timedelta(minutes=1)
Out[7]: datetime.timedelta(0, 60)

In [8]: timedelta(seconds=1 * 60)
Out[8]: datetime.timedelta(0, 60)
```

Reply:
I think I did try that earlier and since it didn't work I switched to `seconds`.  I'll try again and replace.